gradle-language-server/src/main/java/com/microsoft/gradle/utils/LSPUtils.java

```diff
@@ -42,4 +53,14 @@ public static String getStringBeforePosition(String text, Range range, Position
     }
     return text.substring(0, position.getCharacter() - start.getCharacter());
   }
+
+  public static void applySetting(GradleServices services, Object settings) {
```

Comment:
`applySetting` anything to do with LSP?

Reply:
The setting values are converted from LSP Json objects. Maybe something like `GradleServivesUtils` is a better place for this method?

Reply:
to me, LSPUtils should host utilities like converting between LSP objects and other data structures.

Reply:
can it be a public member of class `GradleServices`?

Reply:
> can it be a public member of class `GradleServices`?

Sure.